src/snowflake/snowpark/stored_procedure.py

```diff
@@ -69,6 +70,10 @@ def __call__(
             raise ValueError(
                 f"Incorrect number of arguments passed to the stored procedure. Expected: {len(self._input_types)}, Found: {len(args)}"
             )
+
+        session._conn._telemetry_client.send_function_usage_telemetry(
+            "StoredProcedure.__call__", TelemetryField.FUNC_CAT_USAGE.value
```

Comment:
call this sp or udf will also under `TelemetryField.FUNC_CAT_USAGE`?

Reply:
It will be. Would you recommend a more appropriate category (or a new category?). I am using `create` for udf/sp creation and `usage` for invocations.

Reply:
yea, once a udf is created, it can be called many times. So we should separate them into two different categories

Reply:
Sorry, I don't understand. Is the current implementation ok, or should it be changed to show multiple invocations separately. Currently, create and invoke send different telemetry messages

Reply:
ah I see. I'm not sure how important the category is in telemetry? Because the category is "create", though the message is different, the actual udf invocation has nothing to do with "create"?

Reply:
that's right, invocation has nothing to do with create. We only call create once the stored procedure is registered